blocks/card/card.js

```diff
@@ -0,0 +1,38 @@
+import { createOptimizedPicture } from '../../scripts/aem.js';
+
+export default function decorate(block) {
+  const cardContainer = document.createElement('div');
+  cardContainer.className = 'card-container';
+  
+  const card = document.createElement('div');
+  card.className = 'card';
+
+  const row = block.firstElementChild;
+  if (row) {
+    const [cardImage, cardBody] = row.children;
+
+    if (cardImage) {
+      const img = cardImage.querySelector('img');
+      if (img) {
+        img.className = 'card__image';
+        card.append(img);
+      }
+    }
+
+    if (cardBody) {
+      cardBody.className = 'card__body';
+      const [titleElement, descriptionElement] = cardBody.children;
+      if (titleElement) titleElement.className = 'card__title util-title-s';
+      if (descriptionElement) descriptionElement.className = 'card__description util-body-xs';
+      card.append(cardBody);
+    }
+  }
+
+  card.querySelectorAll('picture > img').forEach((img) => 
```

Comment:
will a card have multiple images? do we need the foreach logic here or can we make it static?